lib/error-message.js

```diff
@@ -42,7 +42,10 @@ function report(options, err, defaultPath) {
     );
   }
 
-  return Anonymize.pathsAndVersions(options, formatHuman(options.debug, err));
+  return Anonymize.pathsAndVersions(
+    options,
+    formatHuman(options?.debug ?? false, err)
```

Comment:
The CLI still supports Node v10, and `?.` and `??` are not supported until v14 :/